examples/index.json

```diff
@@ -2513,6 +2521,30 @@
         "verifyOutput": false,
         "disablePlayground": true,
         "isLearnByExample": false
+      },
+      {
+        "name": "Field Level Caching",
+        "url": "graphql-caching",
+        "verifyBuild": true,
+        "verifyOutput": false,
+        "disablePlayground": true,
+        "isLearnByExample": false
+      },
+      {
+        "name": "Operational Level Caching",
+        "url": "graphql-caching-global",
+        "verifyBuild": true,
+        "verifyOutput": false,
+        "disablePlayground": true,
+        "isLearnByExample": false
+      },
```

Comment:
Shouldn't these two be interchanged? IMO, operation-level caching is the basic way and the field-level is a kinda advanced way.

Reply:
Fixed in [73bf589](https://github.com/ballerina-platform/ballerina-distribution/pull/5188/commits/73bf589184b148f87e83d105386f7ec2710527b7)